packages/generator-electrode/component/templates/packages/component/_package.json

```diff
@@ -28,7 +28,9 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "start": "gulp demo",
-    "test": "gulp check"
+    "test": "gulp check",
+    "format": "prettier<% if (quoteType === "'") { %> --single-quote<% } %> --write \"{src,test}/**/*.{js,jsx}\"",
```

Comment:
The latest electrode generators for component will have a new structure as:
```
-demo-app
   - config
   - src
   - test
- packages
   - component
      - src
      - test
```
Do we need to update the prettier format folders here?